canvas.go

```diff
@@ -13,8 +13,8 @@ type Canvas interface {
 	Focused() Focusable
 
 	Size() Size
-	Scale() float32
-	SetScale(float32)
+	Scale() float64
```

Comment:
This is a breaking api change - anyone accessing this function and using the value could get broken code as a result